Modules/Test/classes/class.ilObjTest.php

```diff
@@ -4926,6 +4930,61 @@ function getUnfilteredEvaluationData()
 					}
 				}
 			}
+			else if($this->isDynamicTest())
```

Comment:
@bheyser Maybe you should move this block (and the other two blocks handling 'random' and 'fixed' tests) to some kind of factory when merging this to fix the referred Mantis issue. I know that my fix/code is not of 'elevated' code quality, but it perfectly fitted to the existing two blocks and hopefully solves the problem ;-). I recommend to check this with your 'conceptional eyes' because I am not very familiar with the CTM.